db/seeds/gobierto_seeds/gobierto_data/recipe.rb

```diff
@@ -122,6 +122,14 @@ def self.run(site)
           }
           license.save
         end
+
+        dataset_default_geometry = site.custom_fields.where(class_name: "GobiertoData::Dataset").find_or_initialize_by(uid: 'gobierto-default-geometry-data-column')
+        if dataset_default_geometry.new_record?
+          dataset_default_geometry.name_translations = { ca: "Recinte", en: "Enclosure", es: "Recinto" }
```

Comment:
Hahahah, what are these names?

Reply:
Also, you need to tell the type of custom field. See in the above code that the query is `site.custom_fields.string.where` (the **.string.** is missing)

Reply:
I though about the field will contains coords or similar, a poligon of leaflet ... an enclosure where values dataset belongs to....
no problem geometry better

Reply:
Sorry, I read "Reciente", I though you copied from a catalan translation.

This field will be used by the maps visualization to fetch the data to show in the map from that column. You can call it "Column to read geometry data"